---
permalink: /publications/
title: "Publications"
toc: true
toc_sticky: true
---

## Scientific Articles

See publication list also at [Google Scholar](http://goo.gl/D5n37X) and [PubMed](https://bit.ly/jmctpubmed).  
  
*Here shown by inverse chronological order*
*#: equal contribution; @: corresponding author*

1. "SPyCi-PDB: A modular command-line interface for back-calculating experimental datatypes of protein structures"  
   Zi Hao Liu, Oufan Zhang, João M. C. Teixeira, Jie Li, Teresa Head-Gordon, Julie D. Forman-Kay  
   **The Journal of Open Source Software**, *2023*, *under review* - [#4861](https://github.com/openjournals/joss-reviews/issues/4861)  

1. "FuzPred: A Web Server for the Sequence-Based Prediction of the Context-Dependent Binding Modes of Proteins"  
   Andras Hatos#, **João M.C. Teixeira#**, Susana Barrera-Vilarmau, Attila Horvath, Silvio CE Tosatto, Michele Vendruscolo, Monika Fuxreiter  
   **Nucleic Acids Research** *2023*, *Accepted*

1. "Alternatively spliced exon regulates context-dependent MEF2D higher-order assembly during myogenesis"  
   Mónika Gönczi, **João M.C. Teixeira**, Susana Barrera-Vilarmau, Laura Mediani, Francesco Antoniani, Tamás Milán Nagy, Zsolt Ráduly, Viktor Ambrus, József Tőzsér, Endre Barta, Katalin E. Kövér, László Csernoch, Serena Carra, Monika Fuxreiter  
   **Nature Communications** *2023*, 14, 1329.  
   <https://doi.org/10.1038/s41467-023-37017-7> (:unlock: open-access)

1. "Learning Correlations between Internal Coordinates to improve 3D Cartesian Coordinates for Proteins"  
   Jie Li, Oufan Zhang, Seokyoung Lee, Ashley Namini, Zi Hao Liu, **João M.C. Teixeira**, Julie D Forman-Kay, Teresa Head-Gordon  
   **Journal of Chemical Theory and Computation** *2023*  
   <https://doi.org/10.1021/acs.jctc.2c01270>  
   [Pre-print @arXiv 2022](https://doi.org/10.48550/arXiv.2205.04676) (:unlock: open-access)  

1. "Protein interactions: anything new?"  
   Barrera-Vilarmau, S; **Teixeira, J. M. C.**; Fuxreiter, M  
   **Essays in Biochemistry** *2022*, 66, 7, 821-830.  
   [https://doi.org/10.1042/EBC20220044](https://pubmed.ncbi.nlm.nih.gov/36416856/) (:unlock: open-access)

1. "IDPConformerGenerator: A Flexible Software Suite for Sampling Conformational Space of Disordered Protein States"  
   **Teixeira, J. M. C.@**; Liu, Z. H.; Namini, A.; Li, J.; Vernon, R. M.; Krzeminski, M.; Shamandy, A. A.; Zhang, O.; Haghighatlari, M.; Yu, L.; Head-Gordon, T.; Forman-Kay, J. D.@  
   **Journal of Physical Chemistry A** *2022*, 126, 5985–6003.  
   <https://doi.org/10.1021/acs.jpca.2c03726> (:unlock: open-access)  
   [Pre-print @bioRxiv](https://doi.org/10.1101/2022.05.28.493726) (:unlock: open-access)  

1. "BioExcel Whitepaper on Scientific Software Development"  
   Abraham, Mark J.; Melquiond, Adrien S.J.; Ippoliti, Emiliano; Gapsys, Vytautas; Hess, Berk; Trellet, Mikael; Rodrigues, João P.G.L.M.; Laure, Erwin; Apostolov, Rossen; de Groot, Bert L.; Bonvin, Alexandre M.J.J.; Lindahl, Erik; Bauer, Paul; **Teixeira, João Correia**; Groenhof, Gerrit; Morozov, Dmitry; Honorato, Rodriga; Jimenez, Brian  
   **Zenodo** *2022*  
   <https://doi.org/10.5281/zenodo.6404474> (:unlock: open-access)

1. "Learning to Evolve Structural Ensembles of Unfolded and Disordered Proteins Using Experimental Solution Data"  
   Zhang, O.; Haghighatlari, M.; Li, J.; **Teixeira, J. M. C.**; Namini, A.; Liu, Z. H.; Forman-Kay, J. D.  
   **arXiv** *2022* *(preprint)*  
   <https://doi.org/10.48550/arXiv.2206.12667> (:unlock: open-access)

1. "Protein Dynamics to Define and Refine Protein Ensembles"  
  Pavithra, N.; Haghighatlari, M.; Ashley, N.; **Teixeira, J. M. C.**; Li, J.; Zhang, O.; Gradinaru, C.; Forman-Kay, J.; Head-Gordon, T.  
  **Journal of Physical Chemistry B** *2022*, 126, 9, 1885-1894.  
   <https://doi.org/10.1021/acs.jpcb.1c10925>  

1. "A Methionine Chemical Shift Based Order Parameter Characterizing Global Protein Dynamics"  
  Chashmniam, S.#; **Teixeira, J. M. C.#**; Paniagua, J. C.; Pons, M.  
  **ChemBioChem** *2021*, 22, 1001–1004.  
   <https://doi.org/10.1002/cbic.202000701>  

1. "PDB-Tools Web: A User-Friendly Interface for the Manipulation of PDB Files"  
  Jiménez‐García, B.; **Teixeira, J. M. C.**; Trellet, M.; Rodrigues, J. P. G. L. M.; Bonvin, A. M. J. J.  
  **Proteins: Structure, Function, and Bioinformatics** *2021*, 89 (3), 330–335.  
   <https://doi.org/10.1002/prot.26018> (:unlock: open-access)  

1. "Taurenmd: A Command-Line Interface for Analysis of Molecular Dynamics Simulations"  
   **Teixeira, J. M. C.@**  
   **Journal of Open Source Software** *2020*, 5 (50), 2175.  
    <https://doi.org/10.21105/joss.02175> (:unlock: open-access)  

1. "Insights on Cross-Species Transmission of SARS-CoV-2 from Structural Modeling"  
  Rodrigues, J. P. G. L. M.; Barrera-Vilarmau, S.#; **Teixeira, J. M. C.#**; Seckel, E.; Kastritis, P.; Levitt, M.  
  **PLOS Computational Biology** *2020*, 16, e1008449.  
   <https://doi.org/10.1371/journal.pcbi.1008449> (:unlock: open-access)  
   [Pre-print @bioRxiv](https://doi.org/10.1101/2020.06.05.136861) (:unlock: open-access)  

1. "Structural Models of Human ACE2 Variants with SARS-CoV-2 Spike Protein for Structure-Based Drug Design"  
  Sorokina, M.#; **Teixeira, J. M. C.#**; Barrera-Vilarmau, S.#; Paschke, R.#; Papasotiriou, I.#; Rodrigues, J. P. G. L. M.#; Kastritis, P. L.#  
  **Scientific Data** *2020*, 7 (1), 309.  
   <https://doi.org/10.1038/s41597-020-00652-6> (:unlock: open-access)  

1. "Extended Experimental Inferential Structure Determination Method in Determining the Structural Ensembles of Disordered Protein States"  
  Lincoff, J.; Haghighatlari, M.; Krzeminski, M.; **Teixeira, J. M. C.**; Gomes, G.-N. W.; Gradinaru, C. C.; Forman-Kay, J. D.; Head-Gordon, T.  
  **Communications Chemistry** *2020*, 3 (74), 1–12.  
   <https://doi.org/10.1038/s42004-020-0323-0>  (:unlock: open-access)  
   [Pre-print @arxiv](https://arxiv.org/abs/1912.12582) (:unlock: open-access)

1. "Cis-Trans Proline Isomers in the Catalytic Domain of Calcineurin"  
  **Teixeira, J. M. C.**; Guasch, A.; Biçer, A.; Aranguren‐Ibáñez, Á.; Chashmniam, S.; Paniagua, J. C.; Pérez‐Riba, M.; Fita, I.; Pons, M.  
  **The FEBS Journal** *2019*, 286 (6), 1230–1239.  
   <https://doi.org/10.1111/febs.14721> (:unlock: open-access)  

1. "A Myristoyl-Binding Site in the SH3 Domain Modulates c-Src Membrane Anchoring"  
  Le Roux, A.-L.; Mohammad, I.-L.; Mateos, B.; Arbesú, M.; Gairí, M.; Khan, F. A.; **Teixeira, J. M. C.**; Pons, M.  
  **iScience** *2019*, 12, 194–203.  
   <https://doi.org/10.1016/j.isci.2019.01.010> (:unlock: open-access)  

1. "Pdb-Tools: A Swiss Army Knife for Molecular Structures"  
  Rodrigues, J. P. G. L. M.; **Teixeira, J. M. C.**; Trellet, M.; Bonvin, A. M. J. J.  
  **F1000Res** *2018*, 7, 1961.  
   <https://doi.org/10.12688/f1000research.17456.1> (:unlock: open-access)  

1. "The Two Isoforms of Lyn Display Different Intramolecular Fuzzy Complexes with the SH3 Domain"  
  **Teixeira, J. M. C.**; Fuentes, H.; Bielskutė, S.; Gairi, M.; Żerko, S.; Koźmiński, W.; Pons, M.  
  **Molecules** *2018*, 23 (11), 2731.  
   <https://doi.org/10.3390/molecules23112731> (:unlock: open-access)  

1. "Farseer-NMR: Automatic Treatment, Analysis and Plotting of Large, Multi-Variable NMR Data"  
  **Teixeira, J. M. C.@**; Skinner, S. P.; Arbesú, M.; Breeze, A. L.; Pons, M.  
  **Journal Biomolecular NMR** *2018*, 71 (1), 1–9.  
   <https://doi.org/10.1007/s10858-018-0182-5> (:unlock: open-access)  

1. "Intramolecular Fuzzy Interactions Involving Intrinsically Disordered Domains"  
  Arbesú, M.; Iruela, G.; Fuentes, H.; **Teixeira, J. M. C.**; Pons, M.  
  **Frontiers Molecular Biosciences** *2018*, 5.  
   <https://doi.org/10.3389/fmolb.2018.00039> (:unlock: open-access)  

1. "The Unique Domain Forms a Fuzzy Intramolecular Complex in Src Family Kinases"  
  Arbesú, M.; Maffei, M.; Cordeiro, T. N.; **Teixeira, J. M. C.**; Pérez, Y.; Bernadó, P.; Roche, S.; Pons, M.  
  **Structure** *2017*, 25 (4), 630-640.e4.  
   <https://doi.org/10.1016/j.str.2017.02.011> (:unlock: open-access)  

1. "An Oxygen-Sensitive Toxin-Antitoxin System"  
  Marimon, O.; **Teixeira, J. M. C.**; Cordeiro, T. N.; Soo, V. W. C.; Wood, T. L.; Mayzel, M.; Amata, I.; García, J.; Morera, A.; Gay, M.; Vilaseca, M.; Orekhov, V. Y.; Wood, T. K.; Pons, M.  
  **Nature Communications** *2016*, 7, 13634.  
   <https://doi.org/10.1038/ncomms13634> (:unlock: open-access)  

1. "A C2HC Zinc Finger Is Essential for the RING-E2 Interaction of the Ubiquitin Ligase RNF12"  
  Bijlmakers, M.-J.; **Teixeira, J. M. C.**; Boer, R.; Mayzel, M.; Puig-Sàrries, P.; Karlsson, G.; Coll, M.; Pons, M.; Crosas, B.  
  **Scientific Reports** *2016*, 6 (1), 29232.  
   <https://doi.org/10.1038/srep29232> (:unlock: open-access)

1. "A Bis(Pyridine N-Oxide) Analogue of DOTA: Relaxometric Properties of the Gd(III) Complex and Efficient Sensitization of Visible and NIR-Emitting Lanthanide(III) Cations Including Pr(III) and Ho(III)"  
  Martins, A. F.; Eliseeva, S. V.; Carvalho, H. F.; **Teixeira, J. M. C.**; Paula, C. T. B.; Hermann, P.; Platas-Iglesias, C.; Petoud, S.; Tóth, E.; Geraldes, C. F. G. C.  
  **Chemistry** *2014*, 20 (45), 14834–14845.  
   <https://doi.org/10.1002/chem.201403856>

1. "Examination of Matrix Metalloproteinase-1 in Solution A PREFERENCE FOR THE PRE-COLLAGENOLYSIS STATE"  
  Cerofolini, L.; Fields, G. B.; Fragai, M.; Geraldes, C. F. G. C.; Luchinat, C.; Parigi, G.; Ravera, E.; Svergun, D. I.; **Teixeira, J. M. C.**  
  **Journal of Biological Chemistry** *2013*, 288 (42), 30659–30671.  
   <https://doi.org/10.1074/jbc.M113.477240> (:unlock: open-access)

1. "The Catalytic Domain of MMP-1 Studied through Tagged Lanthanides"  
  Bertini, I.; Calderone, V.; Cerofolini, L.; Fragai, M.; Geraldes, C. F. G. C.; Hermann, P.; Luchinat, C.; Parigi, G.; **Teixeira, J. M. C.**  
  **FEBS Letters** *2012*, 586 (5), 557–567.  
   <https://doi.org/10.1016/j.febslet.2011.09.020> (:unlock: open-access)

1. "The Interaction of La3+ Complexes of DOTA/DTPA Glycoconjugates with the RCA120 Lectin: A Saturation Transfer Difference NMR Spectroscopic Study"  
  **Teixeira, J. M. C.**; Dias, D. M.; Cañada, F. J.; Martins, J. A.; André, J. P.; Jiménez-Barbero, J.; Geraldes, C. F. G. C.  
  **Journal of Biological Inorganic Chemistry** *2011*, 16 (5), 725.  
   <https://doi.org/10.1007/s00775-011-0773-z>

1. "Enantioselective Binding of a Lanthanide(III) Complex to Human Serum Albumin Studied by 1H STD NMR Techniques"  
  Dias, D. M.; **Teixeira, J. M. C.**; Kuprov, I.; New, E. J.; Parker, D.; Geraldes, C. F. G. C.  
  **Organic & Biomolecular Chemistry** *2011*, 9 (14), 5047–5050.  
   <https://doi.org/10.1039/C1OB05524K>
{: reversed="reversed"}

## Book chapters

1. "Chapter 89: Matrix Metallopeptidase-1/Interstitial Collagenase."  
   **Rawlings - Vol. 1 - Handbook of Proteolytic Enzymes, 4e: Metallopeptidases**, *2023*, *Elsevier*.  
   *(Currently under editorial review)*
   *Invited by Dr Neil D. Rawlings. EMBL-European Bioinformatics Institute alumnus. 34
   Brybank Road, Haverhill, Suffolk, CB9 7WD, UK. Tel: 01440 713859*  

## Newspapers highlights

1. "Entre cães e gatos, galinhas e chimpanzés há animais imunes e susceptíveis ao SARS-CoV-2". [Publico.pt](https://www.publico.pt/2020/11/18/ciencia/noticia/caes-gatos-galinhas-chimpanzes-ha-animais-imunes-susceptiveis-sarscov2-1939100), **2020**
1. "Así es el coronavirus que ha mutado en Dinamarca". [El País](https://elpais.com/ciencia/2020-11-06/asi-es-el-coronavirus-que-ha-mutado-en-dinamarca.html), **2020**
1. "https://www.inverse.com/science/why-some-animals-dont-get-covid". [inverse](https://www.inverse.com/science/why-some-animals-dont-get-covid)
1. "Matrix Metalloproteinases: proteins in action". pNMR news, Paramagnetic NMR Newsletters of the FP7-PEOPLE-2012-ITN “PNMR”. www.pnmr.eu, **2012** ([PDF][paraNMR])

## Teaching

*Workshops I organized and participated as a lecturer or tutor*

1. "Protein docking"  
  **Hands-on Session tutor**  
  EMBL-EBI Structural Bioinformatics virtual course, 11-15th October, *2021*  

1. "Farseer-NMR: automatic treatment, analysis and plotting of large scale, multi-variable BioNMR data – knowledge exchange within the community."  
  **Organizer and speaker**  
  NMR Groups at Utrecht University and Leiden, The Netherlands. *2018*.  
  Hosted by: [Hugo van Ingen][hugo].  
  [Slides](https://drive.google.com/file/d/1tyQMquDDHZA8pUKnYjNUraR3fBe0hgFI/view?usp=sharing) |
  [DOI](https://www.researchgate.net/publication/328108180_Workshop_at_Utrecht_Farseer-NMR_automatic_treatment_analysis_and_plotting_of_large_multi-variable_BioNMR_data)  

1. "Farseer-NMR: automatic treatment, analysis and plotting of large scale, multi-variable BioNMR data."  
  **Organizer and speaker**  
  Parc Cientific de Barcelona, CCiTUB, University of Barcelona, Spain. *2018*.  
  Hosteg by: NMR Facility of the University of Barcelona.  
  [Slides](https://drive.google.com/file/d/1BpIJOeNH6Gxsd0bD6h1jHypfIYaMuD0W/view?usp=sharing) |
  [DOI](https://www.researchgate.net/publication/325796312_Farseer-NMR_automatic_treatment_analysis_and_plotting_of_large_multi-variable_BioNMR_data)  

1. "Farseer-NMR: automatic treatment, analysis and plotting of large scale, multi-variable BioNMR data"  
  **Organizer and speaker**  
  Nuclear Magnetic Resonance Spectroscopy Facility, Faculdade de Ciências e Tecnologia, Universidade Nova de Lisboa, Portugal. *9th May 2018*  
  Hosted by: [Eurico J. Cabrita][ejc]  
  [Slides](https://drive.google.com/file/d/157u0EmmcTUKytak7mKFB0v8ij-bYT3Ip)  

1. "Farseer-NMR: automatic treatment, analysis and plotting of large scale, multi-variable BioNMR data"  
  **Organizer and Speaker**  
  Seminar and Hands-on Session at the Workshop NMR FOCUS entitled: “NMR in health and disease: From Protein-Ligand Interactions to Small Molecules in Tissues”, *7th-11th of May 2018*.  
  [PTNMR][ptnmr], University of Coimbra, Portugal.  
  Hosted by: [Rui Brito][brito]  
  [Slides](https://drive.google.com/file/d/157u0EmmcTUKytak7mKFB0v8ij-bYT3Ip)  

1. "2D- and 3D-NMR Processing and Analysis with introduction to CCPNMR Analysis Suite v2.4."
  **Speaker**  
  Hands-on Session at the Workshop NMR FOCUS entitled: “NMR in health and disease: From Protein-Ligand Interactions to Small Molecules in Tissues”, *7th-11th of May 2018*.  
  [PTNMR][ptnmr], University of Coimbra, Portugal.  
  Hosted by: [Rui Brito][brito]  

1. "Chemical Shift Perturbation – Protein Based Studies"  
  **Speaker**  
  Workshop NMR FOCUS entitled: “NMR in health and disease: From Protein-Ligand Interactions to Small Molecules in Tissues”, *7th-11th of May 2018*  
  [PTNMR][ptnmr], University of Coimbra, Portugal.  

1. "Introductory workshops on Farseer-NMR Software."  
  **Organizer and Speaker**  
  [CIC bioGUNE Institute][cic] of the Bizkaia Technology Park and the University of Basque Country, Spain, *2017*.
  Hosted by: [Jesús Jiménez-Barbero][jjb]  
  [Slides](https://drive.google.com/file/d/1dpwutInEY4BFIva-6IBDjWFn8TgOu6s_)
{: reversed="reversed"}

## Talks

1. "Modular code for a modular software: developing HADDOCK3".  
  [Instruct][instruct] Software Developers Exchange Webinar 5, June 10th, **2022**, *invited*.  
  [Website](https://instruct-eric.eu/content/instruct-software-developers-exchange-webinar-series) |
  [Twitter](https://twitter.com/joaomcteixeira/status/1529025387224768512) |
  [LinkedIn](https://www.linkedin.com/posts/joaomcteixeira_software-developer-exchange-5-modular-activity-6942532657725513728-hi9h/?utm_source=linkedin_share&utm_medium=member_desktop_web) |
  [Youtube](https://www.youtube.com/watch?v=5Uk1EvzCOIg) |
  [Slides](https://docs.google.com/presentation/d/1vW8CtNTLh0UDBjxsfz5f_Td8v-IEX6xY5SExv3SgpPk)  

1. "Introducing HADDOCK3: Enabling modular integrative modelling pipelines"  
  [BioExcel][bioexcel] Webminar, June 7th, **2022**, *invited*  
  [Website](https://bioexcel.eu/webinar-introducing-haddock3-enabling-modular-integrative-modelling-pipelines-2022-05-24/) |
  [Twitter](https://twitter.com/joaomcteixeira/status/1531942717726593025) |
  [LinkedIn](https://www.linkedin.com/posts/joaomcteixeira_bioexcel-webinar-67-introducing-haddock3-activity-6942780130758959104-Losw?utm_source=linkedin_share&utm_medium=member_desktop_web) |
  [Youtube](https://www.youtube.com/watch?v=V7uwFbVDKFE) |
  [Slides](https://docs.google.com/presentation/d/1C8KngBXWAF-6FJkP7DH-j6E1LKwYzqtwS2SSS_g-riM)  

1. "Effective open-science practices for organizing a scientific software repository - extended"  
  International Symposium on Grids & Clouds (ISGC), 20-25th March, **2022**, Virtual Conference.  
  [Abstract](https://docs.google.com/document/d/1dZ4XsI7UahdoSINcNuOiH8T_bZtaHDUMDc1-4qcFDQk) |
  [Slides](https://docs.google.com/presentation/d/1phdI7Q0aiK1dwh7TAcvNqlGVObWLq__WuZ-TUD5XRfM)  

1. "Effective open-science practices for organizing a scientific software repository - short"  
  OSCU Open Science Symposium - Faculty of Science, October 10th, **2021**, Utrecht University, Netherlands.  
  [Slides on OSF](https://osf.io/ghfbr/) |
  [Slides on GDrive](https://docs.google.com/presentation/d/1rD5H7Eps5LHKQAhu21KFywDxbKawOalfXWVDYVVTL0U)  

1. "Modelling Intrinsically Disorder Proteins by experimental data."  
  SickKids Postdoctoral Seminar Series, **2021**, *invited*.  
  [Slides](https://drive.google.com/file/d/1RVlCoFocpvsZp813mqZmGIrJnthC-EFe)  

1. "A summary of the Farseer-NMR project and its current state."  
  Invited Researcher at SickKids, Toronto, Canada, **2019**.  
  Hosted by: [Julie Forman-Kay][jfk]  
  [Slides](https://www.researchgate.net/publication/331629989_A_summary_of_the_Farseer-NMR_project_and_its_current_state)  

1. "Lyn's natural isoforms display different fuzzy complexes in UD-SH3 interaction."  
  Invited Researcher at SickKids, Toronto, Canada, **2019**.  
  Hosted by: [Julie Forman-Kay][jfk]  
  [Slides](https://www.researchgate.net/publication/331629943_Lyn's_natural_isoforms_display_different_fuzzy_complexes_in_UD-SH3_interaction)  

1. Cis-trans proline isomers and ligand binding in the catalytic domain of Calcineurin.  
  Open seminar, RMBLab, Protein Biophysical Chemistry Group, University of Coimbra, Portugal, May 4th, **2018**, *invited*.  
  [Slides](https://drive.google.com/file/d/1iZvg8KxxpR3dy8UahWicgrCaZPSPjJqp)  

1. "Automated NMR data treatment of simple and complex experimental datasets."  
  1 minute flash-presentation at CECAM Workshop: “Disordered protein segments: revisiting the structure-function paradigm”, Paris, France, October, **2017**.  
  [Slide](https://drive.google.com/file/d/1ExXvfZm9URegwFDgGdLPh3uPUB22JJJG)  

1. "The concept of Farseer-NMR: automatic treatment and plotting of lage-scale NMR titration data."  
  Barcelona Biomed Seminars, Parc Scientific de la Universitat de Barcelona, Spain, **2017**.

1. "Analysis of Calcineurin’s structure and conformational changes via methionine labelling NMR spectroscopy."  
  Barcelona Biomed Seminars, Parc Scientific de la Universitat de Barcelona, Spain, February, **2016**.  
  [Slides](https://drive.google.com/file/d/1iZvg8KxxpR3dy8UahWicgrCaZPSPjJqp)  

1. "On the ‘tectonics’ of MMP-1 catalyzed collagenolysis: joint use of paramagnetic NMR and computational tools."  
  Barcelona Biomed Seminars, Parc Scientific de la Universitat de Barcelona, Spain, Februrary, **2014**.  
  [Slides](https://drive.google.com/file/d/1lIPx_kjPaYEjzngZc5vRU04Qd-7ePXmW)

1. "Joint use of NMR and bioinformatic tools to unveil the dynamics and the ‘tectonic’ enzymatic mechanism of MMP-1".  
  EJIBCE, December 20th, **2013**, Oporto, Portugal.  
  [Slides](https://drive.google.com/file/d/1uD-_iTHqtrJLQUpMxnqBYIKH062drlE1)

1. "MMP-1: intradomain mobility and substract interaction."  
  Visiting researcher at the NMR Spectroscopy Research Group, Utrech University, The Netherlands, **2013**.

1. "Molecular Biology preparation of paramagnetic equipped MMP-1."  
  Ph.D. Day 3, University of Florence, Italy, **2012**.

1. "Characterizing MMP-1 Internal Dynamics."  
  IX Annual Meeting of CNC – Center for Neuroscience and Cell Biology, Coimbra, Portugal, **2011**.
{: reversed="reversed"}

## Posters

1. "IDPConformerGenerator: A Flexible Software Suite for Sampling the Conformational Space of Disordered Protein States"  
   67th Biophysical Society Annual Meeting, San Diego CA USA. February 20th 2023, San Diego, USA.  
   [Poster](https://drive.google.com/file/d/1oZsPwGblYuYoyOVXAfQKZFyD1zeES4_O)
1. "Effectively developing, distributing, and managing a Python package with a large community"  
   ISMB/BOSC, July 11-14th, 2022, virtual.  
   [Abstract](https://docs.google.com/document/d/1bKrCh8KTPW-yQ1oPjdY2e-aznNwo-IdE5qpf5OGooUk) |
   [Poster](https://docs.google.com/drawings/d/1hdG7GC4BwFmNrXEH0s4-jexm0cDUPsmOoc8idr7GOTw) |
   [LinkedIn](https://www.linkedin.com/posts/joaomcteixeira_bosc-2022-best-practices-ci-poster-activity-6953258085830291456-UkBu?utm_source=linkedin_share&utm_medium=member_desktop_web)  
1. "Introducing HADDOCK v3": A rebuilt, renewed, modern, extensible, and modular version"  
  Bijvoet Symposium, April 2022. Utrecht, The Netherlands.  
  [Poster](https://docs.google.com/drawings/d/1m7xRsesqtECEgzdQUytcYG6GopRgDSdOWcBSsIXv5s0)
1. "pdb-tools: a swiss army knife for molecular structures"  
  ISMB/ECCB 2019, Basel, Switzerland. F1000 poster publication: https://f1000research.com/posters/8-1173. **Best poster prize**.
1. "Introducing HADDOCK v3": A rebuilt, renewed, modern, extensible, and modular version"  
  EMBO-Workshop: Advances and challenges in biomolecular simulations, October 21st, 2021. Workshop.
  [Poster](https://docs.google.com/drawings/d/1CLHBuumsDkNOdyTixVd93lOUEHUmdKZIDoXGItmqz90)
1. "Automated NMR Data treatment of simple and complex experimental datasets"  
  CECAM Workshop 2017: “Disordered protein segments: revisiting the structure-function paradigm”, Paris, France.  
  [Poster](https://drive.google.com/file/d/1BxqnsPEFGO2Zr3yCHPObbT08CB0i4ydN) 
1. "Farseer - Automatic treatment, calculation and plotting of NMR data. Reducing weeks to seconds"  
  First All Hands Meeting of iNEXT (iNEXT AHM 2016), Alcalá de Henares, Spain.  
  [Poster](https://drive.google.com/file/d/1hfneU3fJWgerhlbspdSptSChvKj5fHs3)
1. "A python tool to automatically treat, calculate and plot NMR-derived peaklists data"  
  16th CCPN / CCPBioSim joint conference 2016, Buxton, United Kingdom.
1. "A python tool to instantly treat, calculate and plot NMR-derived peaklists data"  
  8th GERMN/5th Iberian NMR Meeting 2016, Valencia, Spain. **Awarded Travel Fellowship from GERMN (RSEQ)**.
1. "The interdomain mobility of MMP-1 and its mechanism of interaction with a collagen analogue explored by paramagnetic NMR"  
  5th CAPRI Evaluation Meeting, 17-19 April 2013, Utrecht, The Netherlands.
1. "Analysis of conformational heterogeneity in multidomain enzymes: the prologue of the collagenolysis mechanism"  
  IV Ibero-American NMR Meeting – VI GERMN Bienal Meeting – III Iberian NMR Meeting 2012, Aveiro, Portugal.
1. "3D-view of MMP-1 conformational heterogeneity in solution: the prologue of the collagenolysis mechanism"  
  12th Chianti/INSTRUCT Workshop on BioNMR 2012. Electron and Nuclear Relaxation for Structural Biology, Montecatini Terme, Italy.
1. "3D-view of MMP-1 conformational heterogeneity in solution: the prologue of the collagenolysis mechanism"  
  Breakthroughs in NMR of Structural Biology - 2nd BioNMR Annual User Meeting 2012, Portoroz, Slovenia.
1. "Lanthanide binding CLaNP-5.2 for NMR of proteins: exploring PCS and RDCs in MMP-1 internal dynamics"  
  BioNMR and EAST-NMR Annual User Meeting 2011, Brno, Czech Republic.
1. "Interaction of Ln3+-based MRI contrast agents with HSA using Saturation Transfer Difference NMR Techniques"  
  Joint EUROMAR 2010 and 17th ISMAR conference / Worldwide Magnetic Resonance conference, Florence, Italy.
1. "Study of the interaction of Ln3+ complexes of BOPTA or DTPA-cholate with HSA and of DOTA/DTPA-glycoconjugates with RCA120 using the High Resolution Saturation Transfer Difference (STD) NMR technique"  
  IV Reunión Bienal del GERM-I Reunión Iberica de RMN 2008, Sevilla, Spain.
1. "Fuzzy intramolecular interactions in Src family kinases"  
  EUROMAR 2017, Warsaw,  Poland.
1. "A conserved aromatic-residue code shapes the intrinsically disordered Unique domain of c-Src"  
  16th CCPN / CCPBioSim joint conference 2016, Buxton, United Kingdom.
1. "A conserved aromatic-residue code shapes the intrinsically disordered Unique domain of c-Src"  
  EUROMAR 2016, Aarhus, Denmark.
1. "Transient contacts in the instrinsically Disordered Unique Domain of c-Src"  
  Chemical Complexity & Biology, Strasbourg, France.
1. "A C2HC zinc finger is essential for the activity of the RING ubiquitin ligase RNG125"  
  EMBO Conference 2015 “Ubiquitin and ubiquitin-like modifiers: from molecular mechanisms to human diseases”, Cavtat, Croatia.
1. "Transient contacts in the instrinsically Disordered Unique Domain of c-Src"  
  VII GERMN Biennal Meeting / IV Iberian NMR Meeting / VI Iberoamerican NMR Meeting 2015, Alcalá de Henares, Spain.
1. "Paramagnetic analysis of collagenolysis by matrix metalloproteinase-1 (MMP-1)"  
  EMBO Workshop – Magnetic Resonance for cellular structural biology 2014, Grosseto, Italy.
1. "Interactions of an essential Zn-finger in the ubiquitin ligase RNF125"  
  4th Annual User group Meeting of BioNMR 2014, Warsaw, Poland.
1. "3D-view of MMP-1 conformational heterogeneity in solution: the prologue to the collagenolysis mechanism"  
  XXVth International Conference on Mangetic Resonance in Biological Systems 2012, Lyon, France.
1. "Study of the interaction of La3+ complexes of DOTA/DTPA-glycoconjugates with RCA120 using the High Resolution Saturation Transfer Difference (STD) NMR technique"  
  COST Chemistry D38 “Metal Based Systems for Molecular Imaging Applications” 2009, Warsaw, Poland.
{: reversed="reversed"}


[cic]: https://www.cicbiogune.es/  "CICbioGUNE"
[hugo]: https://www.uu.nl/medewerkers/HvanIngen "Dr Hugo van Ingen"
[jjb]: https://www.cicbiogune.es/people/jjbarbero "Jesús Jiménez-Barbero"
[jfk]: http://biochemistry.utoronto.ca/person/julie-d-forman-kay/  "Dr Julie Forman-Kay"
[ptnmr]: http://ptnmr.web.ua.pt/ "PTNMR"
[brito]: https://www.uc.pt/fctuc/dquimica/rbritolab/members/rmb "Dr Rui Brito"
[ejc]: https://docentes.fct.unl.pt/ejc/ "Dr Eurico J Cabrita"
[instruct]: https://instruct-eric.eu/ "INSTRUCT-ERIC"
[bioexcel]: https://bioexcel.eu/ "BioExcel"
[paraNMR]: https://drive.google.com/file/d/1pX56zTt4AyYNVdooSSJU0mlsq-7tVQQI
